encodings/fsst/src/compress.rs

```diff
@@ -113,7 +118,9 @@ where
     let symbol_lengths_vec: Vec<u8> = compressor.symbol_lengths().to_vec();
     let symbol_lengths =
         PrimitiveArray::from_vec(symbol_lengths_vec, Validity::NonNullable).into_array();
```

Comment:
it's shorter to create non nullable PrimitiveArrays via PrimitiveArray::from(Vec)